core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5401,6 +5409,8 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String WORKER_MASTER_PERIODICAL_RPC_TIMEOUT =
         "alluxio.worker.master.periodical.rpc.timeout";
     public static final String WORKER_MEMORY_SIZE = "alluxio.worker.memory.size";
+    public static final String WORKER_NETWORK_ASYNC_CACHE_MANAGER_WORKQUEUE_MAX =
+        "alluxio.worker.network.async.cache.manager.workqueue.max";
```

Comment:
Is "workqueue" an official term? If not, can we just use `queue.max`?

Reply:
I think it is often used in the context of executor service.

Reply:
but i agree `queue` is shorter and more clear in a generic context